go.mod

```diff
@@ -26,10 +25,12 @@ require (
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.1.0 // indirect
 	go.opentelemetry.io/otel/sdk v1.3.0 // indirect
 	go.opentelemetry.io/proto/otlp v0.10.0 // indirect
-	go.uber.org/goleak v1.1.12 // indirect
 	go.uber.org/zap v1.20.0 // indirect
 	golang.org/x/net v0.0.0-20220105145211-5b0dc2dfae98 // indirect
-	golang.org/x/sys v0.0.0-20211216021012-1d35b9e2eb4e // indirect
 	google.golang.org/genproto v0.0.0-20211223182754-3ac035c7e7cb // indirect
-	google.golang.org/grpc v1.41.0
+	google.golang.org/grpc v1.42.0
+	k8s.io/client-go v0.23.0
+	sigs.k8s.io/controller-runtime v0.11.1
 )
+
+replace github.com/tinkerbell/tink => github.com/micahhausler/tink v0.0.0-20220405214824-8387611ceaea
```

Comment:
What PR's do we depend on to remove this `replace`?

Reply:
https://github.com/tinkerbell/tink/pull/573

Reply:
I can't add labels but if you could add a do-not-merge I'll track the other and ping when its merged.

Reply:
I've added the label.